Tests/PointFreeTests/AccountTests/AccountTests.swift

```diff
@@ -58,7 +58,7 @@ final class AccountIntegrationTests: LiveDatabaseTestCase {
 
     let conn = connection(from: request(to: .team(.leave), session: .loggedIn(as: currentUser)))
 
-    await assertSnapshot(matching: conn |> siteMiddleware, as: .ioConn)
+    await assertSnapshot(matching: await siteMiddleware(conn), as: .conn)
```

Comment:
Wanna find/replace these extra `await`s?

```suggestion
    await assertSnapshot(matching: siteMiddleware(conn), as: .conn)
```